python/src/KarhunenLoeveSVDAlgorithm_doc.i.in

```diff
@@ -24,59 +24,77 @@ centeredSample : bool, default=False
 
-We suppose now that :math:`K < dL`, and we note :math:`\mat{Y} = \sqrt{\mat{W}} \,\mat{X}`.
+We suppose now that :math:`K < Ld`, and we note :math:`\mat{Y} = \sqrt{\mat{W}} \,\mat{X}`.
 
-As the matrix :math:`\mat{\Theta} = \mat{C}` is invertible, the Galerkin and collocation approaches are equivalent and both lead to the following singular value problem for :math:`\mat{Y}`:
+As the matrix :math:`\mat{\Theta} = \mat{C}` is nonsingular, the Galerkin and collocation
+approaches are equivalent and both lead to the following singular value problem for :math:`\mat{Y}`:
 
 .. math::
     :label: QuadCollDim1_ter
 
-     \mat{Y}\,\Tr{\mat{Y}}\,\mat{\Psi}_k  & = \tilde{K} \lambda_k \mat{\Psi}_k
+     \mat{Y}\,\Tr{\mat{Y}}\,\mat{\Psi}_k  & = \widetilde{K} \lambda_k \mat{\Psi}_k.
 
-The SVD decomposition of :math:`\mat{Y}\in \mathcal{M}_{dL,\tilde{K}}(\Rset)` writes:
+The SVD decomposition of :math:`\mat{Y}\in \mathcal{M}_{Ld,\widetilde{K}}(\Rset)` is:
 
 .. math::
 
     \mat{Y} = \mat{U}\, \mat{\Sigma} \, \Tr{\mat{V}}
 
-where  we have :math:`\mat{U} \in \mathcal{M}_{dL,\tilde{K}}(\Rset)`, :math:`\mat{\Sigma}\in \mathcal{M}_{\tilde{K},\tilde{K}}(\Rset)`, :math:`\mat{V} \in \mathcal{M}_{\tilde{K},\tilde{K}}(\Rset)` such that :
+where  the matrices :math:`\mat{U} \in \mathcal{M}_{Ld, \widetilde{K}}(\Rset)`, 
+:math:`\mat{\Sigma}\in \mathcal{M}_{\widetilde{K},\widetilde{K}}(\Rset)`, 
+:math:`\mat{V} \in \mathcal{M}_{\widetilde{K},\widetilde{K}}(\Rset)` are such that :
 
-    - :math:`\Tr{\mat{V}}\mat{V} =\mat{V}\Tr{\mat{V}}= \mat{I}_{\tilde{K}}`,  
-    - :math:`\Tr{\mat{U}}\mat{U} = \mat{I}_{\tilde{K}}` ,
-    - :math:`\mat{\Sigma} = diag(\sigma_1, \dots, \sigma_{\tilde{K}})`.
+- :math:`\Tr{\mat{V}}\mat{V} =\mat{V}\Tr{\mat{V}}= \mat{I}_{\widetilde{K}}`,  
+- :math:`\Tr{\mat{U}}\mat{U} = \mat{I}_{\widetilde{K}}` ,
+- :math:`\mat{\Sigma} = \diag(\sigma_1, \dots, \sigma_{\widetilde{K}})`.
 
-Then the columns of :math:`\mat{U}` are the eigenvectors of :math:`\mat{Y}\Tr{\mat{Y}}` associated to the eigenvalues :math:`\sigma_k^2`.
+Then the columns of :math:`\mat{U}` are the eigenvectors of :math:`\mat{Y}\Tr{\mat{Y}}`
+associated to the eigenvalues :math:`\sigma_k^2`.
 
-We deduce the modes and eigenvalues of the Fredholm problem for :math:`k \leq \tilde{K}`:
+We deduce the modes and eigenvalues of the Fredholm problem for :math:`k \leq \widetilde{K}`:
 
 .. math::
 
-    \begin{align*}
-      \mat{\Phi}_k = \dfrac{1}{\lambda_k} \sqrt{\mat{W}}\, \mat{U}_k
-      \lambda_k = \dfrac{\sigma_k^2}{\tilde{K}}
-    \end{align*}
+      \mat{\Phi}_k & = \dfrac{1}{\lambda_k} \sqrt{\mat{W}}\, \mat{U}_k \\
+      \lambda_k & = \dfrac{\sigma_k^2}{\widetilde{K}}.
 
 We have:
 
 .. math::
 
-    \tilde{\vect{\varphi}}_k(\vect{t})= \sum_{\ell=1}^L  C(\vect{t}, \vect{\xi}_\ell) \vect{\phi}_{\ell,k} \quad \mbox{pour} \quad k \leq \tilde{K}
+    \widetilde{\vect{\varphi}}_k(\vect{t})= \sum_{\ell=1}^L  C(\vect{t}, \vect{\xi}_\ell) \vect{\phi}_{\ell,k} 
```

Comment:
This equation is meaningless. There are 3 sorts of phi functions here: $\phi$, $\varphi$ and $Phi$. Which is which?

Reply:
This is defined in KarhunenLoeveAlgorithm.